addons/udes_stock/models/stock_picking.py

```diff
@@ -627,14 +635,17 @@ def _create_batch(self, pickings):
         PickingBatch = self.env["stock.picking.batch"]
         PickingBatch.create({"picking_ids": [(6, 0, pickings.ids)]})
 
-    def _prepare_move(self, pickings, products_info, **kwargs):
+    def _prepare_move(self, pickings, products_info, extra_product_info_keys=[], **kwargs):
```

Comment:
Is it safe to use a mutable default argument here?

Reply:
Good point: c33d4d9687c19d86e4e8f9d4c4e784fe6a03cb01

Reply:
No longer using this method, will resolve